pyproject.toml

```diff
@@ -37,7 +37,7 @@ dependencies = [
     "click>=8.1.3", # for CLI interface
     "clr-loader==0.2.6",
     "grpcio>=1.30.0",
-    "protobuf>=3.12.2,<3.21.0",
+    "protobuf>=3.12.2",
```

Comment:
Doesn't really matter... the ``ansys-api-mechanical`` package is going to restrict it for you. See my comment there @koubaa